examples/games/game_menu.rs

```diff
@@ -84,10 +84,10 @@ mod splash {
         commands
             .spawn((
                 NodeBundle {
-                    style: Style {
+                    size_constraints: SizeConstraints::FILL_PARENT,
```

Comment:
This example is broken for me in this PR:
![image](https://user-images.githubusercontent.com/2694663/214162282-a5b8450c-ff66-4f2a-9115-3fa652191f14.png)